src/content/en/fundamentals/media/manipulating/applications.md

```diff
@@ -0,0 +1,180 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: An aweful lot of media work requires changing characteristics of media files. In this section, I intend to provide an easy onramp into that world.
+
+{# wf_updated_on: 2017-06-05 #}
+{# wf_published_on: 2017-06-05 #}
+
+# The Applications {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+An aweful lot of media work requires changing characteristics of media files.
+This article covers many things including but not limited to encrypting,
+changing the codec, setting a bit rit, and many others. Finding a
+straightforward way to get started can be bewildering and intimidating. In this
+section, I intend to provide an easy onramp into that world.
+
+You'll find two articles in this section. On this page, I provide some basic
+instruction in using two common media command-line utilities:
+[Shaka Packager](https://github.com/google/shaka-packager) and
+[ffmpeg](https://ffmpeg.org/download.html). Why cover two applications? While
+both are powerful and useful by themselves, neither does everthing needed to
+prepare media for the web.  
+
+The next page is a [cheat sheet](cheatsheet) showing common operations with
+those applications. Corrections and additions to that part are welcome.
+
+
+## Media file characteristics
+
+One thing you'll do often is look at the characteristics of a video: resolution,
+bitrate, codecs, etc.. Since this is just about the easiest thing to do in both
```

Comment:
Actually not. One period for the abbreviation, another for the sentence. I know, looks weird.